dependabot/resources/module_list.json

```diff
@@ -67,6 +67,10 @@
             "name": "module-ballerina-data.xmldata",
             "version_key": "stdlibDataXmldataVersion"
         },
+        {
+            "name": "module-ballerina-data.csv",
+            "version_key": "stdlibDataCsvVersion"
+        },
```

Comment:
Please add the entry according to the alphabetical order

Reply:
Added in https://github.com/ballerina-platform/ballerina-release/pull/2718/commits/bbb4ce2370f42c4fa12c0278fc90e1a20f63adc6